personalwebsite/blog/models.py

```diff
@@ -1,3 +1,33 @@
 from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
+from tinymce.models import HTMLField
 
-# Create your models here.
+
+class Tag(models.Model):
+    name = models.CharField(max_length=56)
+
+    def __str__(self):
+        return self.name
+
+
+class Post(models.Model):
+    title = models.CharField(max_length=256)
+    overview = models.TextField()
+    tags = models.ManyToManyField(Tag)
+    estimated_time = models.IntegerField()
```

Comment:
I think in a future task we can automate this by using some sort of API that estimates reading time-based on word count or make our own. For now, this is fine though,

Reply:
That is a good idea, I will add it to the list.